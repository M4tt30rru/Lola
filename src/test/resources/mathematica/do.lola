##Find do[##Any(_e),{##Identifier(_i),##Literal(_max)}];
 ##replace for(int ##(_i)=0;##(_i)<##(_max);++##(_i)) {##(_e);}

do[f(i);g(i * 2),{i,100}];
