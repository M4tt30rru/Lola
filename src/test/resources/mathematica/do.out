for(int i=0;i<100;++i) {f(i);g(i * 2);}
